### ISSUE TITLE
Request: web2py

### ISSUE THREAD

**Issue by LarsOL** [komarserjio/notejam#43]
In my exploration of the python web frameworks, web2py seems to be the only major one missing.
Is it possible to add an example in this framework?

Thanks even if you can't, these examples are great.

**Comment by komarserjio** [comment]
Yes, it would be nice to have the web2py implementation. I hope some day someone will contribute it :) 
I'm focused on Clojure and Go applications right now. 

I'm happy you enjoy the repo ;)
